src/php/Lang/Tuple.php

```diff
@@ -46,7 +45,7 @@ public static function create(...$values): Tuple
     /**
      * Create a new bracket Tuple.
      *
-     * @param AbstractType|scalar|null ...$values The values
+     * @param AbstractType|scalar|null ...$values
```

Comment:
I never thought about this `AbstractType|scalar|null ...$values`, using the `...`there. But I like it 👍🏼